cves/CVE-2010-0425.yml

```diff
@@ -202,4 +221,14 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: I feel like this vulnerability was due to the developers overlooking
+    it and not quite understanding what was happening in a piece of their code.
+    The fix for the vulnerability was simply deleting 2 lines of code which
+    unloaded the ISAPI.dll module in the middle of a request which created
+    oprhaned callback pointers. Based on this fix it seems that the
+    unloading of the ISAPI.dll method call at that point in the code was
+    completely unecessary. I feel like this vulnerability could have been
+    caught and addressed earlier if the code was tested/reviewed more
+    thoroughly. The biggest takeaway for me from this vulnerability is even
```

Comment:
I think there are probably better takeaways from this vulnerability, as that lesson is pretty generic and kind of obvious. I would instead consider why this vulnerability took so long to be found/fixed.

Reply:
Changed the lessons learned in latest commit.